src/vs/workbench/browser/parts/log/webEmbedderLog.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { LogLevel, LogLevelToString } from 'vs/platform/log/common/log';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { Disposable } from 'vs/base/common/lifecycle';
+import { Extensions, IOutputChannel, IOutputChannelRegistry, IOutputService } from 'vs/workbench/services/output/common/output';
+
+export class WebEmbedderLog extends Disposable {
```

Comment:
I think we had decided that we postpone the decision of where the embedder API implementation lives to May. So even though it is a bit ugly, I would suggest to inline this code into the `web.main.ts` method `window.log` until we have decided how to proceed. That way we at least ensure all the code is in one place and not spread already.

Reply:
I received UI feedback though that logs from the embedder shouldn't be in window.log because it would be too messy and add the potential to be noisy.

I could move the code being done here though into `web.main.ts`?

Reply:
Didn't sandeep clean this up nowß

Reply:
Sorry I was replying as I went down the page. Yes he cleaned it up :)